_maps/RandomRuins/SpaceRuins/derelict6.dmm

```diff
@@ -30,6 +30,7 @@
 /obj/structure/fluff/broken_flooring{
 	icon_state = "pile";
 	dir = 8;
+	
```

Comment:
@SpaceManiac I forgot, what did you want me to do here?

Reply:
`dir = 8;` -> `dir = 8`